doc/user/content/sql/create-source/mysql.md

```diff
@@ -0,0 +1,387 @@
+`TEXT COLUMNS`                       | A list of fully-qualified names | Decode data as `text` for specific columns that contain MySQL types that are unsupported in Materialize.
+
+## Features
+
+### Change data capture
+
+{{< note >}}
+For step-by-step instructions on enabling GTID-based binlog replication for your
+MySQL service, see the integration guides:
+[Amazon RDS](/ingest-data/mysql-amazon-rds/),
+[Amazon Aurora](/ingest-data/mysql-amazon-aurora/),
+[Azure DB](/ingest-data/mysql-azure-db/),
+[Google Cloud SQL](/ingest-data/mysql-google-cloud-sql/),
+[Self-hosted](/ingest-data/mysql-self-hosted/).
+{{< /note >}}
+
+The source uses MySQL's binlog replication protocol to **continually ingest
+changes** resulting from `INSERT`, `UPDATE` and `DELETE` operations in the
+upstream database. This process is known as _change data capture_.
+
+The replication method used is based on [global transaction identifiers (GTIDs)](https://dev.mysql.com/doc/refman/8.0/en/replication-gtids.html),
+and guarantees **transactional consistency** — any operation inside a MySQL
+transaction is assigned the same timestamp in Materialize, which means that the
+source will never show partial results based on partially replicated
+transactions.
+
+Before creating a source in Materialize, you **must** configure the upstream
+MySQL database for GTID-based binlog replication. This requires the following
+configuration changes:
+
+<br>
+
+Configuration parameter          | Value  | Details
+---------------------------------|--------| -------------------------------
+`binlog_format`                  | `ROW`  | This configuration is [deprecated as of MySQL 8.0.34](https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_format). Newer versions of MySQL default to row-based logging.
+`binlog_row_image`               | `FULL` |
+`gtid_mode`                      | `ON`   |
+`enforce_gtid_consistency`       | `ON`   |
+`replica_preserve_commit_order`  | `ON`   | Only required when connecting Materialize to a MySQL read-replica for replication, rather than the MySQL primary.
+
+
+If you're running MySQL as a managed service, further configuration changes
+might be required. For step-by-step instructions on enabling GTID-based binlog
+replication for your MySQL service, see the integration guides.
+
+#### Binlog retention
+
+{{< warning >}}
+If Materialize tries to resume replication and finds GTID gaps due to missing
+binlog files, the source enters an errored state and you have to drop and
+recreate it.
+{{< /warning >}}
+
+By default, MySQL retains binlog files for **30 days** (i.e., 2592000 seconds)
+before automatically removing them. This is configurable via the
+[`binlog_expire_logs_seconds`](https://dev.mysql.com/doc/mysql-replication-excerpt/8.0/en/replication-options-binary-log.html#sysvar_binlog_expire_logs_seconds)
```

Comment:
might be helpful to put `binlog_expire_logs_seconds` in the configuration parameter table above and reference this section under [Details]